ingest-app/src/cmr/ingest/api/ingest.clj

```diff
@@ -361,8 +364,10 @@
 (defn- bulk-update-collections
```

Comment:
I see one here that needs indenting.  There are probably a lot more from your first merge.  I will approve this merge but I think the others should get fixed eventually too.